src/elements/content-uploader/ItemResume.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @flow
+ * @file Item resume component
+ */
+
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PlainButton from '../../components/plain-button/PlainButton';
+import Tooltip from '../../components/tooltip';
+import IconCheck from '../../icons/general/IconCheck';
+import IconRetry from '../../icons/general/IconRetry';
+import LoadingIndicator from '../../components/loading-indicator/LoadingIndicator';
+import { STATUS_PENDING, STATUS_IN_PROGRESS, STATUS_COMPLETE, STATUS_ERROR } from '../../constants';
+import { bdlGreenLight } from '../../styles/variables';
+import messages from '../common/messages';
+
+type Props = {
+    isFolder: boolean,
+    onClick: Function,
+    size: number,
+    status: UploadStatus,
+};
+
+const ItemResume = ({ status, onClick, isFolder = false }: Props) => {
```

Comment:
This file looks very similar to `IconAction`. We should avoid duplicating components when possible.